packages/newsroom-manager/src/Newsroom.tsx

```diff
@@ -65,6 +65,7 @@ export interface NewsroomProps {
   showWalletOnboarding?: boolean;
   showWelcome?: boolean;
   helpUrl?: string;
+  baseUrl?: string;
```

Comment:
Should be `helpUrlBase` or something, same with the props on `<Welcome>` and `<WalletOnboarding>`